sample-apps/java/gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
+
+#
+# Copyright © 2015-2021 the original authors.
```

Comment:
Is this a copyright we can include?

Reply:
Good question for both of these, it is Apache 2.0 so I think it's okay to use, and we would need to keep the original header to use it. Pretty sure these gradle wrappers are normally generated, so we could also switch to have steps for users to generate them themselves